config/initializers/rack_attack.rb

```diff
@@ -28,11 +28,6 @@ class Rack::Attack
     req.ip if req.path =~ paths_regex && req.post?
   end
 
-  # Throttle GET request for api_key by IP address
```

Comment:
Is it removed because that URL doesn't exist anymore?

Reply:
added it back.